README.md

```diff
@@ -5,7 +5,99 @@
 [![LICENSE](https://img.shields.io/github/license/leapfrogtechnology/async-store.svg?style=flat-square)](https://github.com/leapfrogtechnology/async-store/blob/master/LICENSE)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/leapfrogtechnology/async-store)
 
-Global store utility for an async operation lifecycle and chain of callbacks. 
+Global store utility for an async operation lifecycle and chain of callbacks. It is utility tool similar to [continuation-local-storage](https://github.com/othiym23/node-continuation-local-storage) which allows us to set and get values that are scoped to the lifetime of these chains of callbacks. It uses [domain](https://nodejs.org/api/domain.html) native Node.js module.
+
+**Simple example,**
+
+```js
+var asyncStore = require('async-store');
```

Comment:
const

Reply:
and the correct package name it's `@leapfrogtechnology/async-store` check the npm registry link.